microsoft-edge/dualengine/concepts/launching-internet-explorer.md

```diff
@@ -0,0 +1,87 @@
+Engine API.
+
+`-DualEngineVersion=<force_version_number>`
+
+Forces Internet Explorer to instantiate DualEngine objects of the provided version if it can. This will block the 
+creation of DualEngine objects not matching the provided version number, even if this version of Internet Explorer
+supports this version.
+
+## Diagnosing Launch Issues
+There are a number of reasons that Internet Explorer may fail when launching. If this occurs before or while loading
+your DLL it can be difficult to determine the exact reason and communicate it back to your application. Therefore
+if Internet Explorer fails to launch while launching for DualEngine use we have provided a registry value you can 
+check to get detailed error handling information.
+
+### StartupFailurePoint
+`HKCU\SOFTWARE\Microsoft\Internet Explorer\EdgeIntegration\StartupFailurePoint`
+
+Contains a DWORD that indicates the location of the failure in the DualEngine start up path.
+
+The DWORD maps to the following reasons.
+
+| DWORD |   Reason                                                                                      |
+|:-----:|-----------------------------------------------------------------------------------------------|
+|   0   |   No failure                                                                                  |
+|   1   |   Unused                                                                                      |
+|   2   |   Unused                                                                                      |
+|   3   |   The version specified by the DualEngineVersion argument was 0                               |
+|   4   |   Failed to canonicalize the path passed in via DualEngineAdapter, see StartupFailureHresult  |
+|   5   |   Unused                                                                                      |
+|   6   |   Unused                                                                                      |
+|   7   |   LoadLibrary failed for the provided adapter DLL, see StartupFailureHresult                  |
+|   8   |   Could not find DualEngineInitialize in the adapter DLL                                      |
+|   9   |   The call to DualEngineInitialize failed, see StartupFailureHresult                          |
+|   10  |   Failed to set the provided Application User Model ID, see StartupFailureHresult             |
+|   11  |   DLL failed signature check                                                                  |
+|   12  |   Unused                                                                                      |
+|   13  |   DLL did not have IDENTITY LIMITEDACCESSFEATURE resource                                     |
+
+
+### StartupFailureHresult
+`HKCU\Software\Microsoft\Internet Explorer\EdgeIntegration\StartupFailureHresult`
+
+Contains a DWORD that is the failing HRESULT from the StartupFailurePoint if there was one, see above table to

```

Comment:
```suggestion
Contains a DWORD that's the failing HRESULT from the StartupFailurePoint, if there was one.  See the above table to
```

Reply:
Contains a DWORD that's the failing HRESULT from the `StartupFailurePoint`, if there was one.  To see whether a given reason provides a `StartupFailureHresult`, see the above table.